database/buildSchema/22_application_response.sql

```diff
@@ -1,10 +1,16 @@
 -- application response
+CREATE TYPE public.application_response_status AS ENUM (
+    'Draft',
+    'Submitted'
+);
```

Comment:
Since we talked about keeping the Enum strings the same as postGraphile displays them (i.e. "DRAFT", "SUBMITTED"), do you want to start doing that with these new ones? (One less to change later)

Reply:
good point: 2810c55830694ef2560b2d621bcb001f20f2009d

Reply:
btw front end types didn't change (just FYI, global change should be easy from front end point of view)